.github/workflows/stale.yml

```diff
@@ -0,0 +1,28 @@
+name: Stale
+
+on:
+  issues:
+    types: [reopened]
+  schedule:
+    - cron: '*/60 * * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    env:
+      ACTIONS_STEP_DEBUG: true
+    steps:
+      - uses: actions/stale@v3.0.17
```

Comment:
maybe replace that with just `@v3` so it always uses the latest available?

Reply:
done